src/pages/search/SearchPage.tsx

```diff
@@ -58,7 +57,7 @@ const SearchPage = () => {
 
   const requestParams = new URLSearchParams(history.location.search);
   requestParams.delete(SearchParam.Type);
-  const [searchResults, isLoadingSearch] = useFetch<SearchResponse<Registration>>({
+  const [searchResults, isLoadingSearch] = useFetch<RegistrationSearchResponse>({
```

Comment:
Lager en egen type for `RegistrationSearchResponse` for å ha bedre kontroll på typen til aggregeringene